app/views/govuk_web_banners/_emergency_banner.html.erb

```diff
@@ -0,0 +1,11 @@
+<% emergency_banner = GovukWebBanners::EmergencyBanner.new(redis_client: Rails.application.config.emergency_banner_redis_client) %>
```

Comment:
Hmm. I'm in two minds about this, because it doesn't make any difference to the end user, and the utility is debatable for the dev (to me, it makes the partial code 1% nicer, the model code 1% nicer, and the unit tests 25% less readable). But I'm not necessarily the person who's going to do the next change to this code, so if you prefer it I'm okay with adding it. For the moment I've added it in a standalone commit (the last one) so that you can see the changes involved. If we accept it, I'll squash it into the previous commit.

Reply:
I'll have a look at the commit, but in general I'd hide away something so very backend from the frontend bits.